tensorflow_addons/optimizers/README.md

```diff
@@ -10,6 +10,7 @@
 | moving_average | Dheeraj R. Reddy | dheeraj98reddy@gmail.com |
 | rectified_adam | Zhao Hanguang | cyberzhg@gmail.com |
 | weight_decay_optimizers |  Phil Jund | ijund.phil@googlemail.com   |
+| cyclical_learning_rate | Raphael Meudec | raphael.meudec@gmail.com |
```

Comment:
Alphabetize please

Reply:
What do you mean by alphabetize?

Reply:
Just order alphabetically:

Reply:
i.e. place it below conditional gradient in the list

Reply:
Done!